cli/command/container/update.go

```diff
@@ -131,12 +131,12 @@ func runUpdate(ctx context.Context, dockerCli command.Cli, options *updateOption
 
 	var (
 		warns []string
-		errs  []string
+		errs  []error
```

Comment:
it is possible to use errors.Join directly on the error type instead of a slice.

```go
var errs error
errs = errors.Join(errs, err)
errs = errors.Join(errs, err)
```

Reply:
Yeah, they're subtly different though; repeatedly calling `errors.Join` effectively is similar to `errors.Wrap`; the new error is nesting previous errors, whereas a slice approach makes them all "peers"; https://go.dev/play/p/42cXzTSkjoA

```go
package main

import (
	"errors"
	"fmt"
)

func main() {
	errs := []error{
		errors.New("one"),
		errors.New("two"),
		errors.New("three"),
	}
	var nested error
	for _, err := range errs {
		nested = errors.Join(nested, err)
	}

	flat := errors.Join(errs...)

	fmt.Printf("Flat  : %#v\n", flat)
	fmt.Printf("Nested: %#v\n", nested)
}
```